wm/stack.go

```diff
@@ -1,19 +1,25 @@
 package wm
 
-import "fyne.io/desktop"
+import (
+	"fyne.io/desktop"
+	"github.com/BurntSushi/xgb/xproto"
+)
 
 type stack struct {
-	clients []desktop.Window
+	clients      []desktop.Window
+	mappingOrder []desktop.Window
```

Comment:
I don't understand what mappingOrder is here - it doesn't seem like it's part of stack functionality which is reinforced by it requiring the addition of xgb dependency to a high level concept.

Reply:
Extended window manager hints requires two lists... a list of clients based on the order they were added to the stack and a list of clients based on the current position in the stack.  I have removed the xgb dependency.

Reply:
If this truly is a requirement of the stack functionality then it needs appropriate tests. From what you are describing though it seems like an x11 specific requirement for storing mapping order which is quite separate from stacking order.

Reply:
I would imagine other wm specs might want this too... but the simple answer is... stack manages our list of clients - when clients are added when they are removed - etc... it felt like the wrong place to add a client list anywhere else as that was already the functionality of stack.